src/Bicep.Core/Emit/InlineDependencyVisitor.cs

```diff
@@ -128,8 +128,13 @@ private void VisitPropertyAccessSyntaxInternal(PropertyAccessSyntax syntax)
                         return;
                     }
```

Comment:
Is this still needed?

Reply:
Yes - to avoid inlining variables that depend on 'deploy-time' values like `id`, `name`, `type` and `apiVersion`,